src/matcha_ml/cli/destroy.py

```diff
@@ -11,9 +11,12 @@
 from matcha_ml.state import RemoteStateManager
 
 
-def destroy_resources() -> None:
+def destroy_resources(resources: list) -> None:
     """Destroy resources.
 
+    Args:
+        resources(list): the list of resources to be actioned by the verb to be provided to the user as a status message
```

Comment:
missing space between `resource` and `list`. It should be `resources (list)`.

Reply:
Additionally, mypy may flag this as it recommends specifying types of elements inside list such as `List[str]`, etc depending on what this list contains.

Reply:
Thanks Shubham, updated.